help/localSettings.txt

```diff
@@ -1,62 +0,0 @@
-Local developer settings
-========================
-
-The first invocation of any task in Solr gradle build will generate
```

Comment:
I think there is some value in leaving (some content) of this file under help/ - so that folks know that this file can be tweaked and is generated?

Reply:
And keep it wired into `gradlew :helpLocalSettings`?

Reply:
Sorry - I thought I removed  that comment. This is duplicated in the template now so I think it's fine to be removed.

Reply:
Ah bad timing. I can revert that last commit I just put in, or leave it. It might still be a useful signpost